var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -54,10 +55,14 @@ class Tasmanian(CMakePackage):
             description='CMake build type',
             values=('Debug', 'Release'))
 
-    depends_on('cmake@3.5.1:', type='build')
+    depends_on('cmake@2.8:', type='build')
+    depends_on('cmake@3.5:', type='build', when='@6.0:')
+    depends_on('cmake@3.10:', type='build', when='@7.0:')
 
-    depends_on('python@2.7:', when='+python', type=('build', 'run'))
-    depends_on('py-numpy', when='+python', type=('build', 'run'))
+    depends_on('python@2.7:', when='+python @6.0:', type=('build', 'run'))
+    depends_on('python@3.5:', when='+python @7.0:', type=('build', 'run'))
+    depends_on('py-numpy@:1.16', when='+python @:6.0', type=('build', 'run'))
+    depends_on('py-numpy', when='+python @7.0:', type=('build', 'run'))
```

Comment:
I would revert all of these Python/NumPy changes. Currently, you're saying version 5.1 doesn't need NumPy, and can be built with any version of Python.